variables.robot

```diff
@@ -202,6 +203,12 @@ ${OS_UBUNTU}=               ubuntu
 &{RTE55}=                   ip=192.168.10.112
 ...                         platform=minnowboard-turbot
 
+# MPL-PIP4
+&{RTE56}=                   ip=192.168.4.165    cpuid=02c00042ea42e744    pcb_rev=1.0.0
+...                         platform=mpl-pip4    sonoff_ip=192.168.4.222
+...                         board-revision=x    env=dev    platform_vendor=mpl
```

Comment:
Is `env` variables used?
Also others like: `cpuid` `board-revision` ?

Reply:
No, they are not. I think we used other RTE entries as template without much thinking about this. Fixed https://github.com/Dasharo/open-source-firmware-validation/pull/187/commits/29383be17cb72e36e5af37ff8c75c6921cc4ec66